axum-macros/src/from_request.rs

```diff
@@ -421,6 +420,7 @@ fn extract_fields(
     fields: &syn::Fields,
     rejection: &Option<syn::Path>,
     tr: Trait,
+    trait_generics: &Punctuated<Type, Comma>,
```

Comment:
This new paramter doesn't seem to be used.

Reply:
Oh, I guess I was wrong (seeing the revert diff), I probably mistook a `}` to be a level higher. Anyways, if the simpler version works, let's go with that.